README.md

```diff
@@ -64,32 +54,35 @@ Add the following:
 Rails.application.config.assets.paths << Rails.root.join("node_modules", "bitstyles", "bitstyles")
 ```
 
-If you’re using a version of Rails previous to 4, you must instead add the path to the asset pipeline in `config/application.rb`:
+### Importing the sass to your project
+
+For the simplest case where you don’t need to add any CSS component classes, you can import bitstyles and be done:
 
-```ruby
-config.assets.paths << Rails.root.join("node_modules", "bitstyles", "bitstyles")
+```scss
+@import '~bitstyles/scss/bitstyles.scss';
 ```
 
-### Install an unpublished version
+#### Override variables
 
-If you need to install a version of bitstyles that’s so far unpublished (e.g. there’s a bugfix or feature that’s not in a published package yet), you can use the following:
+It’s very likely that you’ll need to set your own colors and fonts. Do this by importing your variables before you import bitstyles:
 
-```sh
-yarn add git+https://github.com/bitcrowd/bitstyles.git#master
+```scss
+@import 'path/to/local-overrides.scss';
+@import '~bitstyles/scss/bitstyles.scss';
 ```
 
-### Importing the sass to your project
+For the complete list of global variables you can override, look through the various files in the `bitstyles/settings/` folder.
+
+Each atom, molecule, organism, and utility class also have their own variables that can be overridden, to be found in the `settings.scss` file in the respective component’s folder.
 
-Copy the contents of the bitstyles manifest file (`bitstyles.scss`) to your project’s own sass manifest file (e.g. `app.scss`, `main.scss`) to make use of the entire library. Most likely you’ll not need everything in there, so comment-out the lines of objects you don’t need:
+#### Adding your own components
+
+If you need to add your own custom component CSS (i.e. not a component built using the utility classes), you’ll need to insert your components in the middle of bitstyles. This is because CSS is a source order-dependent language.
```

Comment:
```suggestion
If you want to add your own custom component CSS (i.e. not a component built using the utility classes), you’ll need to insert your components in the middle of bitstyles. This is because CSS is a source order-dependent language.
```

There is a lot of "needing" here 😉 

Also that part isn't fully obvious to me:
> in the middle of bitstyles

Does it mean "as part of bitstyles"?

Reply:
Great feedback, thanks — I’ll update this section to be clearer